data/human/human missions.txt

```diff
@@ -5733,3 +5733,227 @@ mission "Argosy Hijacking (Declined)"
+			`	For a long while, the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room brilliantly bathed in blue light. On the walls, there are all kinds of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
+			choice
+				`	(Go up to the sword.)`
+					goto sword-scene
+				`	(Take a look around first.)`
+			`	You walk around to the right of the dome to the collection of alien objects, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
+			`	Now that Liam points it out, many of the larger artifacts over his way look similar to the Sheragi technology you studied with Albert Foster.`
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`	After some time, Sophie calls you both to come to look at the tablet.`
+			choice
+				`	(Go over to Sophie.)`
+			label sword-scene
+			`	Approaching the tablet, there is a short message written on it:`
+			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
+			`			"But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
+			`			"This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
+			`			"Your father."`
+			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`
+			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought through about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`
```

Comment:
I don't understand why they think the sword is different from the other artifacts. Perhaps some kind of explanation is needed?

Reply:
I will think of a way to improve this paragraph.

Reply:
Okay, in the original PR there was a picture of the sword inserted in its pedestal, quite prominent over everything else. I think the reason I rewrote it this way was because the siblings believed that:
a. since their father had this sword by their side during his explorations, it should continued to be used or else "it would be wasted" on them
b. they couldn't take and study everything, given their limited resources

The thing about `a` is that the dialogue doesn't actually mention if the siblings know this, just that they say the sword has been. It is a narrative gap that I haven't thought about yet. A better explanation might be needed, but I will reread the whole thing and see if it really is needed as you've been the first to point this out.

Reply:
I thought that they give you the sword because they don't get into situations where they could ever plausibly use it. If they keep it, it's kind of wasted on them since it'll never get used. This is why they give it to the player, who actually does get into fights and other situations where it could be useful.

Reply:
Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

Reply:
> Ah, I see! I suppose I just thought there would be more to what the sword does than just chopping things. :-)

In fact, I did suggest more as well; I had an idea that whenever the sword was swung, an energy bolt like out of an Energy Blaster would be emitted. However, describing that in the dialogue would be clunky, and I don't think it's worth some dialogue and an event to update the sword.

> Plus, wouldn't they want to put an ancient artifact in a museum rather than have it destroyed through use? But I'm obviously just overthinking things.

Emerald Sword.